ludwig/modules/lr_scheduler.py

```diff
@@ -83,8 +106,18 @@ def eval_step(self, progress_tracker: ProgressTracker, validation_field: str):
         last_metric: TrainerMetric = split_metrics[validation_field][validation_metric][-1]
         last_metric_value = last_metric[-1]
 
+        prev_num_reductions = self._eval_scheduler.num_reduce_lr
         self._eval_scheduler.step(last_metric_value)
 
+        num_reductions = self._eval_scheduler.num_reduce_lr
+        if num_reductions > prev_num_reductions:
+            # LR reduction -> update progress tracker
+            progress_tracker.last_learning_rate_reduction_steps = progress_tracker.steps
+            progress_tracker.last_learning_rate_reduction = 0
+            progress_tracker.num_reductions_learning_rate += 1
+        else:
+            progress_tracker.steps - progress_tracker.last_learning_rate_reduction_steps
```

Comment:
Should this be: `progress_tracker.last_learning_rate_reduction_steps = progress_tracker.steps - progress_tracker.last_learning_rate_reduction_steps`?

Reply:
Ah, good catch!